### DISCUSSION TITLE
How to install when running 2 versions of OBS?

### DISCUSSION THREAD

**Discussion by RoyceRemix** [Xaymar/obs-stream-effects#994] [Q&A] [answered]
I have 2 non-portable versions of OBS (v27 & v28) installed and I tried installing StreamFX, but it installed to ProgramData automatically, rather than a specific obs directory...  I don't see the filters in obs and I'm not quite sure what I've done wrong - I also don't really understand if it matters whether I choose installing for all users or only myself (I'm the only user on my PC)

**Comment by Xaymar** [comment, marked as answer]
Only Portable supports multiple installations. Even obs itself doesn't have support for multiple static installations.
